### ISSUE TITLE
cleanup docker compose and fullnode/validator templates to avoid operator confusion

### ISSUE THREAD

**Issue by geekflyer** [aptos-labs/aptos-core#2435]
As discussed in https://aptos-org.slack.com/archives/C034W35ARFX/p1659318103411649 the fullnode+validator docker configs and config files could use some love and cleanup.

Specifically:
- consider removing docker/compose/public_full_node in favor of aptos-node
- consider removing docker/compose/validator-testnet in favor of `aptos node run-local-testnet --with-faucet`
- rename docker/compose/aptos-node/docker-compose.yaml to docker-compose-validator.yaml
- consider increasing consistency with between validator/node config files. At the moment we have various config files spread throughout the repo and it's not clear what's intentional vs accidental differences between some of them.
 - In general make the intention of each compose or node config file template clear.
 - Update the docs accordingly

**Comment by ilhanu** [comment]
The current [instructions](https://aptos.dev/nodes/full-node/fullnode-source-code-or-docker#approach-2-using-docker) for docker on devnet don't work for me on ubuntu 20.04 and 22.04 with latest docker installed using the convenience script.

**Comment by hairangsua** [comment]
> 

the same as me. i follow step by step in documents but stuck.

it alway show this error

_Node should start correctly: DB failed to bootstrap status INVALID_WRITE_SET of type Validation_

**Comment by geekflyer** [comment]
@ilhanu  @hairangsua  thanks for reporting. I did a quick fix to update the instructions here https://github.com/aptos-labs/aptos-core/commit/58125560a47ab1bde9bca464956c8e8a74a7582f . The cause appears that we published yesterday already docker images for the next devnet release, without updating our DevNet validators to that version. With my fix the instructions will download a specific version of the docker image that corresponds to the DevNet release from last week which is currently active.
